src/openfermion/measurements/_hamiltonian_stabilizers_sets.py

```diff
@@ -0,0 +1,93 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""Tools to split the Hamiltonian into subsets based on the support of
+its stabilizers."""
+
+import numpy
+import copy
+from openfermion.ops import QubitOperator
+
+
+def _has_disjoint_indices(op1, op2):
+    """
+    Auxiliary function of get Hamiltonian subsets.
+
+    Check if two QubitOperators have any qubit in common.
+
+    Args:
+        op1(QubitOperator): Single Pauli string.
+        op2(QubitOperator): Single Pauli string.
+
+    Returns:
+        overlap (Boolean): True if the operator and stabilizer overlap.
```

Comment:
This needs to be updated. "True if the operators have disjoint indices, False otherwise."